GOVERNANCE.md

```diff
@@ -64,10 +64,6 @@ A guide for Collaborators is maintained in
 ## CTC Membership
 
 CTC seats are not time-limited.  There is no fixed size of the CTC.
-However, the expected target is between 6 and 12, to ensure adequate
-coverage of important areas of expertise, balanced with the ability to
-make decisions efficiently.
```

Comment:
I'm restoring the original language and just removing the part about target size. We can alter the original language, I suppose, but hopefully it can be agreed that that would be a separate issue from simply removing the target sizes.